.travis.yml

```diff
@@ -100,25 +100,39 @@ jobs:
        - popd
     - name: "presubmit"
       env:
-       - PRESUB_TESTS=true GOFLAGS='-race' PRESUBMIT_OPTS="--no-linters --no-generate"
+       - GOFLAGS='-race' PRESUBMIT_OPTS="--no-linters --no-generate" GO_TEST_TIMEOUT=20m
```

Comment:
You can put these one line versions all on the same line like so:

```
script: ./scripts/presubmit.sh ${PRESUBMIT_OPTS}
```

Same for env

Reply:
Done.